clusters/base/kube-prometheus-stack.yaml

```diff
@@ -8,7 +8,7 @@ spec:
   interval: 24h
   url: https://prometheus-community.github.io/helm-charts
 ---
-apiVersion: helm.toolkit.fluxcd.io/v2beta1
+apiVersion: helm.toolkit.fluxcd.io/v2beta2
```

Comment:
I don't think this will work. We need to upgrade Flux I created https://github.com/cncf-tags/green-reviews-tooling/issues/54 for that

Reply:
thanks :+1: